# Move base node parameters. For full documentation of the parameters in this file, please see
#
#  http://www.ros.org/wiki/move_base
#
shutdown_costmaps: false

controller_frequency: 10.0
controller_patience: 3.0

planner_frequency: 2.0
planner_patience: 5.0

oscillation_timeout: 10.0
oscillation_distance: 0.2

#######################
# Planner Definitions
#######################

# Local Planner
base_local_planner: "teb_local_planner/TebLocalPlannerROS"

## Local Planner Options:
# dwa_local_planner/DWAPlannerROS
# base_local_planner/TrajectoryPlannerROS
# eband_local_planner/EBandPlannerROS
# teb_local_planner/TebLocalPlannerROS

base_global_planner: "runners_planner/RunnersPlanner"

## Global Planner Options:
# global_planner/GlobalPlanner
# navfn/NavfnROS
# carrot_planner/CarrotPlanner
# runners_planner/RunnersPlanner

#######################
# Recovery Behaviour
#######################

recovery_behavior_enabled: true
clearing_rotation_allowed: false

name: "conservative_reset"
type: "clear_costmap_recovery/ClearCostmapRecovery"
reset_distance: 1.5

name: 'aggressive_reset'
type: 'clear_costmap_recovery/ClearCostmapRecovery'
reset_distance: 3.0

